microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -35,9 +35,12 @@ See also:
 
 
 <!-- ====================================================================== -->
-## Install WinAppSDK
+## Install WinAppSDK and enable Developer Mode
 
-To enable developing widgets on your local machine, install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
+To enable developing and testing widgets on your local machine:
+
+* Install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
+* Enable Developer Mode in Windows 11: go to **Settings** > **For developers** and then enable **Developer Mode**.
```

Comment:
Should this be 
```suggestion
* Enable Developer Mode in Windows 11: go to **Settings** > **Privacy & security** >  **For developers** and then enable **Developer Mode**.
```

Reply:
Oddly, I don't have the option under **Settings** > **Privacy & security**. I have it under **Settings** > **System**. And I also have it on the **Settings** landing page (but maybe that default list is built from settings I've accessed recently).

Reply:
Nevermind, the **Settings** app opens up _on_ the **System** tab. And that's where I see **For developers**.